api/WindowOrWorkerGlobalScope.json

```
@@ -570,10 +570,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/WindowOrWorkerGlobalScope/crossOriginIsolated",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "87"
             },
             "chrome_android": {
-              "version_added": false
+              "version_added": "87"
             },
             "edge": {
               "version_added": false
```

suggestion: mirror to other Chromium-derived browsers

According to the [intent to ship](https://groups.google.com/a/chromium.org/g/blink-dev/c/cJ5dXIcQCsc) and some direct testing of my own, this shipped on all of these browsers except WebView. Can we set Edge to 87 and Opera (desktop) to 73?

```suggestion
              "version_added": "87"
```

---

Thanks for the suggestion @ddbeck - I have updated the PR now.